nrf52-hal-common/src/gpio.rs

```diff
@@ -62,7 +62,8 @@ use crate::target::P0;
 #[cfg(feature = "52840")]
 use crate::target::{ P1 };
 
-use crate::hal::digital::{OutputPin, StatefulOutputPin, InputPin};
+use crate::hal::digital::v2::{OutputPin, StatefulOutputPin, InputPin};
+use core::convert::Infallible;
```

Comment:
Does using this increase our MSRV? IIRC this came in in 1.34.

Reply:
I don't know what the MSRV is, but if `Infallible` needs to be replaced, it should be replaced with `void::Void`, which is already a dependency of this crate.

Reply:
Awesome, I went ahead and replaced `Infallible` with `void::Void`.